src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillApiTransport.php

```diff
@@ -97,6 +97,10 @@ private function getPayload(Email $email, Envelope $envelope): array
             $payload['message']['subaccount'] = $email->getHeaders()->get('X-MC-Subaccount')->getBodyAsString();
         }
 
+        if ($email->getHeaders()->get('X-MC-Tags')) {
```

Comment:
I think either it should pass all the headers into the request or there should be an abstraction of sort. Putting the raw version defined by the provider, in my opinion, means we're leaking implementation details. And if we can't or shouldn't put an abstraction around it because it's a single provider, then I think the user of the library should be aware of the implementation details, not the library.